cli/Cargo.toml

```
@@ -14,6 +14,7 @@ path = "src/main.rs"
 [dependencies]
 iota-sdk = { path = "../sdk", default-features = false, features = [ "wallet", "tls", "storage", "rocksdb", "stronghold", "participation" ] }
 
+chrono = { version = "0.4.24", default-features = false, features = [ "std" ]}
```

```suggestion
chrono = { version = "0.4.24", default-features = false, features = [ "std" ] }
```
Sorry I forgot this space in my suggestion, my bad